api/buildout.cfg

```diff
@@ -1,6 +1,6 @@
 [buildout]
 index = https://pypi.python.org/simple
-extends = http://dist.plone.org/release/5.1rc1/versions.cfg
```

Comment:
I have some tests running also on 5.1 on travis: https://github.com/RedTurtle/collective.analyticspanel/blob/master/.travis.yml

The trick is to pass a python version and when you run buildout, not using "/bin/buildout" but only "buildout".
If i remember correctly, travis automatically activates the virtualenv.

Reply:
@cekk we can pass a python version in travis. This would fix the issue. Though, this is a node project and travis does not allow you to have two languages at the same time.

Reply:
good point..ok, nevermind ;)

Reply:
I guess we could put more effort into it to figure it out. Though, maybe just waiting for Travis to upgrade to Ubuntu 16.04 will do as well. :)